examples/ale_classification.ipynb

```diff
@@ -0,0 +1,787 @@
+{
```

Comment:
<p>another way to lay these out would be:</p><ul><li>each row is the same target</li><li>each col is the feature</li></ul><p>I think that could be a bit less sore on the eye but I don't mind the current version too much as it is simple enough</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>I considered that way the first time round, I ended up going with this because the number of plots is independent of the number of targets. In any case, targets can be switched off to focus on any subset of classes.</p>